subjects/0-shell/scripting/README.md

```diff
@@ -0,0 +1,75 @@
+## scripting
+
+### Objectives
+
+You must follow the same [principles](https://public.01-edu.org/subjects/0-shell/) as the first subject.
+
+In `scripting` you must find a way to make your `0-shell` run `bash` scripts.
```

Comment:
Bash is too complex and has too many advanced structures to be taken as a example.
`ash` instead ? (integrated in BusyBox and in Debian as `dash`)